src/util/misc.ts

```diff
@@ -173,25 +173,25 @@ export function saveStaticFiles(config: Config, zimCreator: ZimCreator) {
   ]);
 }
 
-export function cssPath({ output: { dirs } }: Config, css: string) {
-  return [dirs.style, `${dirs.styleModules}/${css.replace(/(\.css)?$/, '')}.css`].join('/');
+export function cssPath(css: string, subDirectory = '') {
+  return `${subDirectory ? `${subDirectory}/` : ''}${css.replace(/(\.css)?$/, '')}.css`;
 }
-export function jsPath({ output: { dirs } }: Config, js: string) {
+export function jsPath(js: string, subDirectory = '') {
   const path = (isNodeModule(js)) ? normalizeModule(js) : js;
-  return [dirs.javascript, `${dirs.jsModules}/${path.replace(/(\.js)?$/, '')}.js`].join('/');
+  return `${subDirectory ? `${config.output.dirs.mediawiki}/` : ''}${path.replace(/(\.js)?$/, '')}.js`;
 }
-export function genHeaderCSSLink(config: Config, css: string, articleId: string, classList = '') {
+export function genHeaderCSSLink(config: Config, css: string, articleId: string, subDirectory = '', classList = '') {
   const resourceNamespace = '-';
   const slashesInUrl = articleId.split('/').length - 1;
   const upStr = '../'.repeat(slashesInUrl + 1);
-  return `<link href="${upStr}${resourceNamespace}/${cssPath(config, css)}" rel="stylesheet" type="text/css" class="${classList}" />`;
+  return `<link href="${upStr}${resourceNamespace}/${cssPath(css, subDirectory)}" rel="stylesheet" type="text/css" class="${classList}" />`;
 }
-export function genHeaderScript(config: Config, js: string, articleId: string, classList = '') {
+export function genHeaderScript(config: Config, js: string, articleId: string, subDirectory = '', classList = '') {
```

Comment:
typechecking? For `classlist` as well?

Reply:
sure....but one thing I wanted to point out I didn't see class list being passed anywhere while calling these function will we need them at some point

Reply:
then please remove it.